core/process.py

```diff
@@ -1196,7 +1196,7 @@ def __init__(self):
   #
   # [job_id, flag, pgid, job_state, node]
 
-  def NotifyStopped(self, pid):
+  def WhenStopped(self, pid):
     # type: (int) -> None
```

Comment:
this one also needs its type comment updated

Reply:
will do.